code/controllers/subsystem/ticker.dm

```diff
@@ -422,13 +422,12 @@ var/datum/subsystem/ticker/ticker
 	var/list/total_antagonists = list()
 	//Look into all mobs in world, dead or alive
 	for(var/datum/mind/Mind in minds)
```

Comment:
Do you mean use a for loop which casts directly to the assumed type? If so then it might not be needed since this is called only once.